Unsupported URL Type "link:": link:./resources/eslint-rules

Getting the following on :-
```
git clone https://github.com/graphql/graphql-js.git
cd graphql-js
npm install
```
on both Windows and Mac OSX with latest NPM.

```
C:\Users\aaron\GitHub\graphql-js>npm install
npm ERR! code EUNSUPPORTEDPROTOCOL
npm ERR! Unsupported URL Type "link:": link:./resources/eslint-rules

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\aaron\AppData\Roaming\npm-cache\_logs\2020-04-23T17_49_59_928Z-debug.log
```
C:\Users\aaron\AppData\Roaming\npm-cache\_logs\2020-04-23T17_49_59_928Z-debug.log
```
0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'install'
1 verbose cli ]
2 info using npm@6.14.4
3 info using node@v14.0.0
4 verbose npm-session 0c652f051e4fd10e
5 silly install runPreinstallTopLevelLifecycles
6 silly preinstall graphql@15.0.0
7 info lifecycle graphql@15.0.0~preinstall: graphql@15.0.0
8 silly install loadCurrentTree
9 silly install readLocalPackageData
10 timing stage:loadCurrentTree Completed in 10ms
11 silly install loadIdealTree
12 silly install cloneCurrentTreeToIdealTree
13 timing stage:loadIdealTree:cloneCurrentTree Completed in 0ms
14 silly install loadShrinkwrap
15 timing stage:loadIdealTree:loadShrinkwrap Completed in 1ms
16 silly install loadAllDepsIntoIdealTree
17 timing stage:rollbackFailedOptional Completed in 1ms
18 timing stage:runTopLevelLifecycles Completed in 39ms
19 silly saveTree graphql@15.0.0
20 verbose stack Error: Unsupported URL Type "link:": link:./resources/eslint-rules
20 verbose stack     at unsupportedURLType (C:\Program Files\nodejs\node_modules\npm\node_modules\npm-package-arg\npa.js:200:15)
20 verbose stack     at fromURL (C:\Program Files\nodejs\node_modules\npm\node_modules\npm-package-arg\npa.js:258:13)
20 verbose stack     at Function.resolve (C:\Program Files\nodejs\node_modules\npm\node_modules\npm-package-arg\npa.js:77:12)
20 verbose stack     at C:\Program Files\nodejs\node_modules\npm\lib\install\deps.js:489:20
20 verbose stack     at Array.map (<anonymous>)
20 verbose stack     at C:\Program Files\nodejs\node_modules\npm\lib\install\deps.js:488:47
20 verbose stack     at tryCatcher (C:\Program Files\nodejs\node_modules\npm\node_modules\bluebird\js\release\util.js:16:23)
20 verbose stack     at Promise._settlePromiseFromHandler (C:\Program Files\nodejs\node_modules\npm\node_modules\bluebird\js\release\promise.js:517:31)
20 verbose stack     at Promise._settlePromise (C:\Program Files\nodejs\node_modules\npm\node_modules\bluebird\js\release\promise.js:574:18)
20 verbose stack     at Promise._settlePromiseCtx (C:\Program Files\nodejs\node_modules\npm\node_modules\bluebird\js\release\promise.js:611:10)
20 verbose stack     at _drainQueueStep (C:\Program Files\nodejs\node_modules\npm\node_modules\bluebird\js\release\async.js:142:12)
20 verbose stack     at _drainQueue (C:\Program Files\nodejs\node_modules\npm\node_modules\bluebird\js\release\async.js:131:9)
20 verbose stack     at Async._drainQueues (C:\Program Files\nodejs\node_modules\npm\node_modules\bluebird\js\release\async.js:147:5)
20 verbose stack     at Immediate.Async.drainQueues [as _onImmediate] (C:\Program Files\nodejs\node_modules\npm\node_modules\bluebird\js\release\async.js:17:14)
20 verbose stack     at processImmediate (internal/timers.js:456:21)
21 verbose cwd C:\Users\aaron\GitHub\graphql-js
22 verbose Windows_NT 10.0.18363
23 verbose argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install"
24 verbose node v14.0.0
25 verbose npm  v6.14.4
26 error code EUNSUPPORTEDPROTOCOL
27 error Unsupported URL Type "link:": link:./resources/eslint-rules
28 verbose exit [ 1, true ]
```

--- a/package.json
+++ b/package.json
@@ -56,7 +56,7 @@
     "dtslint": "3.6.0",
     "eslint": "7.0.0",
     "eslint-plugin-flowtype": "4.7.0",
-    "eslint-plugin-graphql-internal": "link:./resources/eslint-rules",
+    "eslint-plugin-graphql-internal": "file:./resources/eslint-rules",
     "eslint-plugin-import": "2.20.2",
     "eslint-plugin-node": "^11.1.0",
     "flow-bin": "0.124.0",

